src/Illuminate/Cache/Repository.php

```diff
@@ -490,7 +490,7 @@ protected function getMinutes($duration)
             $duration = Carbon::now()->diffInSeconds(Carbon::instance($duration), false) / 60;
         }
 
-        return $duration > 0 ? $duration : null;
+        return (int) ($duration * 60) > 0 ? $duration : null;
     }
```

Comment:
This means you cannot cache things for less than one minute. See for example https://laracasts.com/discuss/channels/general-discussion/caching-less-than-1-minute?page=1